src/Client.js

```diff
@@ -274,38 +274,13 @@ export default class Client {
 
     const transaction = await this.getMethod('getTransactionByHash')(txHash)
 
-    const valid = this.validateTransaction(transaction)
+    if (transaction) {
+      const valid = this.validateTransaction(transaction)
 
-    if (!valid) {
-      const errors = this.validateTransaction.errors
-      throw new InvalidProviderResponseError(`Provider returned an invalid transaction: ${errors[0].dataPath} ${errors[0].message}`)
-    }
-
-    return transaction
-  }
-
-  /**
```

Comment:
Not sure I agree with removing it just because it doesn't have an Ethereum implementation. Bitcoin developers may have a specific use case for it. Although since `getTransactionByHash` already has it, I think it makes sense 👍

Reply:
The client should only have cross chain methods otherwise it can become really cluttered. 
Solely bitcoin devs wouldn't find much use in CAL I think and if they wanted to use a bitcoin only implemented function, they can always use the provider directly.